libsignal-service/src/push_service.rs

```diff
@@ -255,25 +255,9 @@ pub struct SignalServiceProfile {
     pub about_emoji: Option<Vec<u8>>,
 }
 
-#[derive(Debug, serde::Deserialize, Default)]
-#[serde(rename_all = "camelCase")]
-pub struct CdnUploadAttributes {
-    path: String,
-    acl: String,
-    key: String,
-    policy: String,
-    algorithm: String,
-    credential: String,
-    date: String,
-    signature: String,
-    content_type: String,
-    length: u64,
-}
-
```

Comment:
Is this unused now?

Reply:
Seems to be, yes. I cleaned up compilation warnings in the PR.